### PR TITLE
feat: Don't hide the settings tab when an instance is running

### LINKED ISSUE #854: Don't hide the settings tab when an instance is running.
### Role

I use PrismLauncher 😳

### Suggestion

Don't hide the settings tab when an instance is running

### Benefit

Less confusing

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

I don't see why the settings tab should get hidden while an instance is running, instead all buttons inside it should be greyed out/non interactable like they are for most other tabs.

Non running Instance

running Instance

### PR DESCRIPTION
**Pull request by RedsonBr140**
Before, when an instance was launched, the "Settings" tab in the edit menu would just disappear. Now everything in this tab just gets greyed out.

https://github.com/PrismLauncher/PrismLauncher/assets/61910521/e9dbb4f7-6ad8-433b-8650-584da4dd7dfc

closes #854

### CODE REVIEW

**Review by flowln — APPROVED** [review]
thx!

File: launcher/ui/pages/instance/InstanceSettingsPage.cpp [hunk 1]
```diff
@@ -74,7 +78,7 @@ InstanceSettingsPage::InstanceSettingsPage(BaseInstance *inst, QWidget *parent)
 
 bool InstanceSettingsPage::shouldDisplay() const
 {
-    return !m_instance->isRunning();
+    return true;
 }
```
Comment (TheKodeToad) [inline launcher/ui/pages/instance/InstanceSettingsPage.cpp]: It should behave the same if you remove it altogether.

Comment (RedsonBr140) [reply launcher/ui/pages/instance/InstanceSettingsPage.cpp]: So... Should I remove this entire function and every call?

Comment (TheKodeToad) [reply launcher/ui/pages/instance/InstanceSettingsPage.cpp]: Oh, it's a virtual function and it's already implemented by BasePage so no need to remove calls (in fact that would not be what you want)

**Comment by ghost** [comment]
You probably shouldn't even grey out many of those options, as they'd only apply on next startup anyway.

**Comment by RedsonBr140** [comment]
> You probably shouldn't even grey out many of those options, as they'd only apply on next startup anyway.

Idk -- it's better than just hiding the entire tab I guess

**Comment by ghost** [comment]
yeah

**Review by TheKodeToad — APPROVED** [review]
`updateRunningStatus` is pretty much the same as `setDisabled`. I think this is more logical though - the action may change!

**Review by getchoo — APPROVED** [review]
this has always bugged me a little :p

**Comment by getchoo** [comment]
and don't forget to rebase 😄

**Comment by TheKodeToad** [comment]
I did it ;3 I hope that's fine

**Comment by getchoo** [comment]
looks like we might have our first merge for 8.0 👀

**Comment by TheKodeToad** [comment]
:woman_shrugging: wouldn't 7.1 make more sense?
edit: flow did this before 7.0 was released

**Comment by getchoo** [comment]
that is a good point actually. i don't see how this is much different from #1063 🤔

**Review by DioEgizio — APPROVED** [review]

**Pull request merged**